.rubocop.yml

```diff
@@ -35,7 +35,7 @@ Metrics/ClassLength:
   Max: 1000
 
 Metrics/AbcSize:
-  Max: 100
+  Max: 150
```

Comment:
150 is really high for AbcSize (IIRC default is 15).  If we have a method that's violating we should probably fix or disable the check for that method via # rubocop:disable Metrics/AbcSize.